phoenix-core/src/main/java/org/apache/phoenix/parse/AlterIndexStatement.java

```diff
@@ -65,6 +67,10 @@ public boolean isAsync() {
         return async;
     }
 
+    public boolean isAll() {
```

Comment:
Name could be a little confusing. Maybe "isRebuildAll", with a similar rename for the variable?

Reply:
I can do that. But the reason I picked this name is; this class maps directly to Sql statement. The name of the keyword is ALL

Reply:
I agree isAll is confusing, please rename it to isRebuildAll. Currently it only applies to rebuild.